api/internal/managers/kubernetes/installation/config.go

```diff
@@ -19,6 +19,29 @@ func (m *installationManager) SetConfig(config types.KubernetesInstallationConfi
 	return m.installationStore.SetConfig(config)
 }
 
+// GetDefaults returns the default values for Kubernetes installation configuration
+func (m *installationManager) GetDefaults() (types.KubernetesInstallationConfig, error) {
+	defaults := types.KubernetesInstallationConfig{
+		AdminConsolePort: ecv1beta1.DefaultAdminConsolePort,
+	}
+
+	return defaults, nil
+}
+
+// SetConfigDefaults sets default values for the installation configuration
+func (m *installationManager) SetConfigDefaults(config *types.KubernetesInstallationConfig) error {
+	defaults, err := m.GetDefaults()
+	if err != nil {
+		return fmt.Errorf("get defaults: %w", err)
+	}
+
+	if config.AdminConsolePort == 0 {
+		config.AdminConsolePort = defaults.AdminConsolePort
+	}
```

Comment:
Should the defaults also take into account the proxy setting? Similar to what we do for linux:
- https://github.com/replicatedhq/embedded-cluster/blob/a0b2de5234a5fc497df1935aedaa26ea226c3f93/api/internal/managers/linux/installation/config.go#L190
- https://github.com/replicatedhq/embedded-cluster/blob/a0b2de5234a5fc497df1935aedaa26ea226c3f93/pkg-new/config/proxy.go#L128-L143

Reply:
Done via - https://github.com/replicatedhq/embedded-cluster/pull/2788/commits/ae6fe6212c042a874fc49e2f633baca11eaccbce